src/uu/dd/src/progress.rs

```diff
@@ -183,7 +194,10 @@ impl ProgUpdate {
     /// assert_eq!(iter.next().unwrap(), b"");
     /// assert!(iter.next().is_none());
     /// ```
-    fn write_transfer_stats(&self, w: &mut impl Write) -> std::io::Result<()> {
+    fn write_transfer_stats(&self, w: &mut impl Write, new_line: bool) -> std::io::Result<()> {
```

Comment:
The documentation and the example code should be updated for this new parameter.

Reply:
Thanks, done.